packages/desktop-client/src/components/accounts/Balance.jsx

```diff
@@ -188,6 +189,11 @@ export function Balances({
             marginLeft: 10,
             color: theme.pillText,
             transform: showExtraBalances ? 'rotateZ(180deg)' : 'rotateZ(0)',
+            opacity: isButtonHovered
+              ? 1
+              : selectedItems.size > 0 || showExtraBalances
+                ? 1
+                : 0,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
            opacity: isButtonHovered || selectedItems.size > 0 || showExtraBalances
                ? 1
                : 0,
```

Is there no way to do this with just CSS anymore? Doing it with CSS would be more performant.

Reply:
I was considering using emotion but since we have been trying to move to a newer framework, I just thought to inline it for now and revisit once we replace emotion.